### PR TITLE
fix(axios-ext): adds additional log for deployment

### PR DESCRIPTION
**Pull request by cianmSAP**
#1257

Adds an additional log at the end to clarify the URL may require an update to point to the deployed app

### CODE REVIEW

File: .changeset/heavy-socks-rhyme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/axios-extension': minor
```
Comment (zdravko-georgiev) [inline .changeset/heavy-socks-rhyme.md:2]: Do we really need `minor` version bump?

Comment (cianmSAP) [reply .changeset/heavy-socks-rhyme.md]: Updated, thanks @zdravko-georgiev

**Review by zdravko-georgiev — APPROVED** [review]
Change is simple and easy to understand
Review comments addressed
Excellent test coverage
Did NOT tested manually

**Review by devinea — APPROVED** [review]
Change looks appropriate and covered by test.
Changeset ✅

**Pull request merged**